### ISSUE TITLE
'stddef.h' file not found

### ISSUE THREAD

**Issue by jmwright** [CadQuery/pywrap#6]
run.sh seems to complete, but I see a lot of errors similar to this earlier in the output.
```
[D 191211 08:42:02 module:23] ./opencascade/IFSelect_SelectSignature.hxx
[W 191211 08:42:02 header:36] ./opencascade/IFSelect_SelectSignature.hxx
[W 191211 08:42:02 header:37] /usr/lib/gcc/x86_64-linux-gnu/7.4.0/../../../../include/c++/7.4.0/cstddef:50:10: fatal error: 'stddef.h' file not found
[D 191211 08:42:02 module:23] ./opencascade/IFSelect_SelectUnknownEntities.hxx
[W 191211 08:42:02 header:36] ./opencascade/IFSelect_SelectUnknownEntities.hxx
[W 191211 08:42:02 header:37] /usr/lib/gcc/x86_64-linux-gnu/7.4.0/../../../../include/c++/7.4.0/cstddef:50:10: fatal error: 'stddef.h' file not found
[D 191211 08:42:03 module:23] ./opencascade/IFSelect_EditForm.hxx
[W 191211 08:42:03 header:36] ./opencascade/IFSelect_EditForm.hxx
[W 191211 08:42:03 header:37] /usr/lib/gcc/x86_64-linux-gnu/7.4.0/../../../../include/c++/7.4.0/cstddef:50:10: fatal error: 'stddef.h' file not found
```
I tried installing clang since that seemed to be how most people solved a similar error, but that didn't help. @adam-urbanczyk Have you encountered this?

**Comment by adam-urbanczyk** [comment]
Take a look here:

https://github.com/CadQuery/pywrap/blob/e0815bbdd85928f82434294cfcfb44466d98888f/bindgen/header.py#L20

I assumed that this Include path specification will be universal enough. Seems that I'm wrong. Not sure how to solve this nicely but you need to add the path to relevant includes here.

**Comment by jmwright** [comment]
I'm not sure why my environment has a different version of clang, but it does. I was able to fix it by adding the following line.
```
args.append('-I{}'.format(Path(sys.prefix) / 'lib/clang/6.0.1/include/'))
```
As far as I can tell, the build completes properly now.

**Comment by adam-urbanczyk** [comment]
Solved in 8e8635f6920793eb26b7a786cad0ea1d9ec726dc thanks for the patience @jmwright .
